### PR TITLE
impelement 1st year/01 Basic Calculator python solution

### PR DESCRIPTION
**Pull request by VickyGuo0907**
### Have you read the [Contributing Guidelines on Pull Requests]()?

- [X] Yes

### Description

Add Python code for 1st year/ 01 Basic Calculator 

### Checklist

- [X] I've read the contribution guidelines.
- [X] This PR meets all the requirements like Clean Code, proper README and Documentation.

### Related Issues or Pull Requests
Issue: #5 

### Screenshots (if any)

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot** | **updated screenshot**|

### CODE REVIEW

**Comment by VickyGuo0907** [comment]
@atarax665 Somehow I closed this pr by accident. Please review my first PR.

**Comment by VickyGuo0907** [comment]
@atarax665  Would you mind review this PR?

File: 1st Year/01. Basic Calculator/calculator.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import operator
```
Comment (atarax665) [inline 1st Year/01. Basic Calculator/calculator.py:1]: Do not use inbuilt functions, these programs are for learning purpose, traditional way is preferred.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: 1st Year/01. Basic Calculator/calculator.py (hunk 1 above)
Comment (VickyGuo0907) [reply 1st Year/01. Basic Calculator/calculator.py]: Updated implementation.

**Review by atarax665 — APPROVED** [review]
Perfect :+1:

**Pull request merged**